ufld/CMakeLists.txt

```diff
@@ -12,6 +12,11 @@ find_package(CUDA REQUIRED)
 
 include_directories(${PROJECT_SOURCE_DIR}/include)
 
+# cuda_runtime_api
+include_directories(/usr/local/cuda-10.2/targets/aarch64-linux/include)
```

Comment:
Here is only for aarch64, we need remove these 2 lines.

Reply:
fine，I have removed them.